locale/id_ID/author.xml

```diff
@@ -1,122 +1,118 @@
+	<message key="author.submissions.queuedEditingCopyedit">Dalam Pengeditan: Perlu Copyedit</message> 
+	<message key="author.submissions.queuedEditingProofread">Dalam Pengeditan: Perlu Proofread</message>
+	<message key="author.submissions.uploadCopyeditedVersion">Unggah Versi Pasca-Copyedit</message> 
+	<message key="author.submissions.proofreadingCorrections">Koreksi Proofreading</message>	
+	<message key="author.submissions.confirmDelete">Apakah Anda yakin akan menghapus naskah yang belum selesai ini?</message>	
+	<message key="author.submissions.viewStatus">Lihat Status</message>
+	<message key="author.submissions.noSubmissions">Tidak ada penyerahan naskah</message> 
+	<message key="author.submit.journalSection">Bagian Jurnal</message>
+	<message key="author.submit.journalSectionDescription">Naskah harus diserahkan ke salah satu bagian jurnal.</message>
+	<message key="author.submit.verifyChecklist">Anda harus memastikan semua artikel tercantum dalam daftar nama naskah yang memuaskan sebelum dilanjutkan.</message>	
+	<message key="author.submit.copyrightNoticeAgree">Penulis setuju dengan persyaratan hak cipta, yang akan berlaku pada naskah ini saat diterbitkan oleh jurnal (komentar untuk editor dapat ditambahkan di bawah).</message>
+	<message key="author.submit.copyrightNoticeAgreeRequired">Anda harus menyetujui persyaratan hak cipta sebelum melanjutkan</message>
+	<message key="submission.submit.coverNote">Komentar untuk Editor</message> 
+	<message key="author.submit.comments">Masukkan Teks (pilihan)</message> 
+	<message key="author.submit.form.sectionRequired">Pilih bagian jurnal yang sesuai untuk penyerahan ini.</message>	
+	<message key="author.submit.form.localeRequired">Pilih bahasa penyerahan naskah.</message> 
+	<message key="submission.submit.privacyStatement">Pernyataan Privasi Jurnal</message>
+	<message key="author.submit.reorderInstructions">Mengurutkan nama penulis agar ditampilkan seperti urutan dalam terbitan.</message> 
+	<message key="author.submit.reorder">Mengurutkan nama penulis</message> 
+	<message key="author.submit.selectPrincipalContact">Kontak Utama untuk Korespondensi Editorial.</message>	
+	<message key="author.submit.addAuthor">Tambah Penulis</message> 
+	<message key="author.submit.deleteAuthor">Hapus Penulis</message> 
+	<message key="author.submit.submissionIndexingDescription">Masukkan istilah untuk mengindeks naskah; pisahkan istilah dengan titik koma (term1; term2; term3).</message> 
+	<message key="author.submit.coverageInstructions">Mengacu pada lokasi geo-spasial, liputan atau cakupan kronologis atau historis, dan/ atau karakteristik sampel penelitian.</message> 
+	<message key="author.submit.languageInstructions"><![CDATA[English=en; French=fr; Spanish=es. <a href="http://www.loc.gov/standards/iso639-2/php/code_list.php" target="_blank">Kode Tambahan</a>.]]></message>	<message key="author.submit.form.authorRequired">Setidaknya dibutuhkan satu penulis.</message>	

```

Comment:
Are you sure you want to leave this English string in here? If someone is looking for missing keys in a translation, they will not discover this…